tests/ignite/metrics/nlp/test_bleu.py

```diff
@@ -148,7 +177,7 @@ def _test_distrib_integration(device):
 
     data = []
     for c in corpus.chunks:
-        data += idist.get_world_size() * [c]
+        data += idist.get_world_size() * [([c[0]], [c[1]])]
```

Comment:
I would prefer modifying the corpus.chunks and keep this as the original. Adding batch in the corpus.chunks would be necessary to validate batch + ddp.